securedrop/source_templates/lookup.html

```diff
@@ -67,7 +67,7 @@ <h2 class="headline">Get Replies</h2>
       <div class="reply">
         <time class="date" title="{{ reply.date|datetimeformat }}" datetime="{{ reply.date }}">{{ reply.date|datetimeformat(relative=True) }}</time>
         <form id="delete" class="message" method="post" action="/delete" autocomplete="off">
-          <input name="csrf_token" type="hidden" value="{{ csrf_token() }}"/>
+          <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
           <input type="hidden" name="reply_filename" value="{{ reply.filename }}" autocomplete="off"/>
           <a href="#confirm-delete-{{ reply.filename }}" class="delete">
```

Comment:
Side note, noticed while reviewing: this `<a>` start tag appears to be missing a corresponding closing tag.